pyanaconda/ui/tui/spokes/installation_source.py

```diff
@@ -376,14 +359,14 @@ def apply(self):
             self._nfs_server = self._nfs_server[6:]
 
         try:
-            (self.data.method.server, self.data.method.dir) = self._nfs_server.split(":", 2)
+            host, path = self._nfs_server.split(":", 2)
         except ValueError as err:
             log.error("ValueError: %s", err)
             self._error = True
             return
 
         opts = self._nfs_opts or ""
-        self.set_source_nfs(opts)
+        self.set_source_nfs(opts=opts, host=host, path=path)
```

Comment:
This have to change to:
```suggestion
        self.set_source_nfs(server=host, directory=path, opts=opts)
```

https://github.com/rhinstaller/anaconda/pull/2533/commits/486b8e094a5a0044ad0e765cf9df8f64c53be8f2#diff-b4c6badd586849c76c60e56fc1a215e9R141

Reply:
Fixed.